public/content/lessons/2017/eulers-formula-via-group-theory/index.mdx

```diff
@@ -1,9 +1,215 @@
+
+## What are Exponents?
+
+So that’s the primer on groups. Groups are symmetric actions on some mathematical object, whether that’s a square, a circle, the real number line, or anything else you dream up. Every group has a certain arithmetic, where you can combine two actions by applying one after the other, and asking what other action from the group gives the same overall effect.
+
+Numbers, both real and complex, can be thought of in two different ways as a group. They can act by sliding, in which case the group arithmetic looks just like ordinary addition. Or they can act by these stretching/squishing/rotating actions, in which case the group arithmetic looks just like multiplication.
+
+Now let’s talk about exponentiation. Our first introduction to exponents is to think of them in terms of repeated multiplication, right? The meaning of something like $2^3$ is to take $2\cdot 2\cdot 2$, and the meaning of $2^5$ is $2\cdot 2\cdot 2\cdot 2\cdot 2$.
+
+A consequence of this, something you might call the exponential property, is that if I add two numbers in the exponent, say $2^{3+5},$ this can be broken up into a product $2^3\cdot 2^5$. 
+
+But expressions like $2^{1/2}$ or $2^{-1}$, and much less $2^i$, don’t really make sense if you think of exponents as repeated multiplication. What does it mean to multiply 2 by itself half of a time, or $-1$ of a time?
+
+So we do something that’s very common throughout math, and extend the definition to apply to exponents beyond counting numbers. But we don’t just do this randomly, if you think back to how fractional and negative exponents are defined, it’s always motivated by trying to make sure this property $2^{x+y}$ = $2^x 2^y$ still holds.
+
+To see what this might mean for complex exponents, think of what this property is saying from a group theory perspective. It’s saying that adding the inputs corresponds with multiplying the outputs, and that makes it very tempting to think of the inputs not merely as numbers, but as members of the additive group of sliding actions, and to think of the outputs not merely as numbers, but as members of the multiplicative group of stretching and squishing actions.
+
+<Figure image="add_multiply_homo.png" width="60%" />
+
+It’s weird and strange to think about a function that takes in one kind of action and spits out another kind of action, but that’s actually something which comes up all the time in group theory. This exponential property is very important for this association between groups, it forms a map between the additive and multiplicative group for real numbers.
+
+This exponentiation function guarantees that if I compose two sliding actions, maybe a slide by $-1$, then a slide by $2$, it corresponds to composing the output actions, in this case squishing by $2^{-1}$, then stretching by $2^2$.
+$$
+\begin{align*}
+2^{-1+2}&=2^{-1}\cdot2^2 \\
+\exp(-1+2)&=\exp(-1)\cdot\exp(2)

```

Comment:
I think I would move the exponential function down to the number e section and talk about preserving group structure there. For someone who hasn't seen the exponential function before, this could potentially be confusing.